src/snackbar/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { WidgetBase } from '@dojo/framework/widget-core/WidgetBase';
+import { tsx } from '@dojo/framework/widget-core/tsx';
+import I18nMixin from '@dojo/framework/widget-core/mixins/I18n';
+import Button from '../button/index';
+import * as css from '../theme/snackbar.m.css';
+import bundle from './nls/Snackbar';
+
+export interface SnackbarProperties {
+	success?: boolean;
```

Comment:
are there any more plans for different states in the future? would this make sense to be a `type` property and string literal types rather than a boolean?

Reply:
Good point! I changed this to be an optional `type?: 'success' | 'error'` property, so it can be extended later.